packages/mathtype-tinymce5/README.md

```diff
@@ -36,9 +37,6 @@ Easily include quality math equations in your documents and digital content.
    ```js
    tinymce.init({
     toolbar: 'tiny_mce_wiris_formulaEditor tiny_mce_wiris_formulaEditorChemistry',
-    // Necessary
-    htmlAllowedTags: ['.*'],
-    htmlAllowedAttrs: ['.*'],

```

Comment:
Is deleting these lines an error or is it on purpose?

Reply:
It's on purpose.  😃 

Probably you missed, but I've added a comment on the "More information" section of this pull request's description. 

I believe that this `htmlAllowedTags` option was added to the TinyMCE5 instructions, by mistake.

I didn't found any reference to this option on the TinyMCE documentation site and a simple google search by the exact term only points out to the Froala website.

See: https://froala.com/wysiwyg-editor/docs/options/#htmlAllowedTags

In any case, I coudn't resist to fix it since it only affected documentation and, even though I am sure it's not causing any bug on the client, is adding unnecessary clutter to the technical documentation.